src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Use constants for the number of days that define long-term rental and the corresponding discount value, to avoid hardcoding literals in expressions (checklist item 2).